py-geopolars/python/geopolars/internals/geodataframe.py

```diff
@@ -1,23 +1,53 @@
-import polars
+from __future__ import annotations
 
+import polars as pl
+
+from geopolars import geopolars as core  # type: ignore
 from geopolars.internals.geoseries import GeoSeries
 
 DEFAULT_GEO_COLUMN_NAME = "geometry"
 
 
-class GeoDataFrame(polars.DataFrame):
+class GeoDataFrame(pl.DataFrame):
 
     _geometry_column_name = DEFAULT_GEO_COLUMN_NAME
 
     def __init__(self, data=None, columns=None, orient=None, *, geometry=None):
 
         # Wrap an existing polars DataFrame
-        if isinstance(data, polars.DataFrame):
+        if isinstance(data, pl.DataFrame):
             self._df = data._df
             return
 
         super().__init__(data, columns, orient)
 
+    def get_column(self, name: str) -> pl.Series | GeoSeries:
```

Comment:
I've always used `Union[...]` instead of `A | B`. Can you remind me what's the first Python version that supports `A | B`? I had thought it was higher than 3.7, which we currently support.

Reply:
Hmm so it looks like you _can_ use it in Python 3.7 annotations, but you can never let Python evaluate the types (it will only be readable by mypy). https://mypy.readthedocs.io/en/stable/runtime_troubles.html#using-x-y-syntax-for-unions

polars seems to use `|` extensively, and in general I'd try to align with their style, so I'd say this is ok 👍